### ISSUE TITLE
network structure

### ISSUE THREAD

**Issue by huangnengCSU** [taki0112/ResNeXt-Tensorflow#6]
the structure seems to be different from the ResNeXt block in the paper "Aggregated Residual Transformations for Deep Neural Networks".

**Comment by AILearnerLi** [comment]
I agree with huangnengCSU. In my opinion, this  just is resnet,but not resnext. Because in the split_layer, it is not group convolution.

**Comment by KhurramPirov** [comment]
> I agree with huangnengCSU. In my opinion, this just is resnet,but not resnext. Because in the split_layer, it is not group convolution.

Did you find the right implementation with splits, e.g version b from the article?

**Comment by whuhxb** [comment]
@huangnengCSU  @KhurramPirov  @AILearnerLi  @taki0112 

This implementation seems to be resnet-inception, not resnext. Did you have any link to realize the resnext?
